devchat.go

```diff
@@ -1171,3 +1245,10 @@ func remove(s []*user, a *user) []*user {
 	}
 	return s
 }
+
+type styleApplication func(string) string
```

Comment:
Nah you don’t really need this new type, let the definition of color styles directly use a function